firmware/application/usb_serial_shell.cpp

```diff
@@ -907,6 +907,191 @@ static void cmd_cpld_read(BaseSequentialStream* chp, int argc, char* argv[]) {
     }
 }
 
+static uint32_t reverse(uint32_t x) {
```

Comment:
Isn't there a __builtin_bitreverse32 in our gcc ?
If not, wouldn't a lookup table approach be quicker ?